R/wflow_toc.R

```diff
@@ -0,0 +1,45 @@
+#' Add a table of content on the clipboard
+#'
+#' \code{wfow_toc} add a table of content on the clipboard. The users can paste it
+#' in the \code{index.Rmd} document. The table of content includes all \code{html} document
+#' in the \code{docs} directory the users want to display.
+#'
+#' @param gh_url character. The Github Pages root directory.
+#' @param proj_url character. The project name.
+#'
+#' @return text character. The list of table of content in the markdown syntax.
+#' @import fs
+#' @import glue
+#' @import dplyr
+#' @import purrr
+#' @import readr
+#' @import tidyr
+#' @import clipr
+#' @import here
+#' @import rmarkdown
+#' @import tibble
+#' @export
+
+wflow_toc <- function(analysis_dir = "analysis", docs_dir = "docs") {
+  s <- wflow_status()
```

Comment:
After adding `project = "."` as the default argument to `wflow_toc()`, you can change `wflow_status()` to `wflow_status(project = project)`.

Reply:
LGTM, I will do it.

Reply:
I edit it, here, https://github.com/jdblischak/workflowr/pull/155/commits/8787eedb8db24ed51620f206765db6c2985e280f